rskj-core/src/main/java/co/rsk/rpc/netty/http/modules/HealthCheckModule.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2018 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package co.rsk.rpc.netty.http.modules;
+
+import co.rsk.rpc.netty.http.HttpUtils;
+import io.netty.handler.codec.http.DefaultFullHttpResponse;
+import io.netty.handler.codec.http.HttpMethod;
+
+public class HealthCheckModule implements HttpModule {
+
+    @Override
+    public DefaultFullHttpResponse processRequest(String uri, HttpMethod method) {
+        if ("/health-check/ping".equals(uri) && method.equals(HttpMethod.GET)) {
+            return HttpUtils.createResponse("pong");
```

Comment:
Maybe this PR is just the first step of a health-check but... shouldn't we be checking things like enabled modules, transactions and blocks being recorded correctly, etc?